### PR TITLE
Home

### PR DESCRIPTION
**Pull request by md-amir-gauhar**
-> added home screen
-> added modal form
Screenshot (195)
Screenshot (196)

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import '../styles/Home.css'
+
+const Home = () => {
+  const [name, setName] = useState("Amir")
+  return (
+    <div className='home'>
+      <h1 className="welcome">
+        Welcome Back, <span className='name'>{name}</span>!
+      </h1>
+      <h3 className='todosleft'>You have {3} tasks for today. All the best!</h3>
```
Comment (znjs) [inline src/pages/Home.js:12]: didn't understand the purpose of wrapping 3 in `{}`

**Review by znjs — APPROVED** [review]
Good folder structure

**Review by Pratik1005 — APPROVED** [review]
Code is clean and easy to understand 👍

File: src/pages/Home.js (hunk 1 above)
Comment (md-amir-gauhar) [reply src/pages/Home.js:12]: I forgot to put todos.length there😩. Thanks for pointing it out.

**Pull request merged**